server/ControlPlane/Buffers/Buffers.cs

```diff
@@ -137,6 +138,7 @@ public static void MapBuffers(this RouteGroupBuilder root)
                 var count = await manager.SoftDeleteBuffers(tagQuery, excludeTagQuery, ttl, purge, cancellationToken);
                 return Results.Ok(count);
             })
+            .RequireOwnerRole()
```

Comment:
Could this pattern come and bite us with later functionality. Seems like `.RequireOwnerRole()` should be the default and `.AllowContributorRole()` is the opt-in? Perhaps too much hassle. But one could imagine us adding some restricted functionality in the future and forgetting this.

Reply:
I agree with you. Good point.